python/ray/dag/compiled_dag_node.py

```diff
@@ -2132,11 +2132,20 @@ def execute(
 
         if self._returns_list:
             ref = [
-                CompiledDAGRef(self, self._execution_index, channel_index)
+                CompiledDAGRef(
+                    self,
+                    self._execution_index,
+                    list(self.worker_task_refs.values()),
```

Comment:
I think you do not need to pass these in explicitly since you can access them through `self`